airbyte-cli/build.gradle

```
@@ -1,3 +1,22 @@
+import com.bmuschko.gradle.docker.tasks.image.DockerBuildImage
+
 plugins {
-    id "airbyte-docker"
+    id 'com.bmuschko.docker-remote-api'
 }
+
+task copyDocker(type: Copy) {
+    delete "build/docker"
+
+    from "${project.projectDir}/Dockerfile"
+    into "build/docker/"
+}
+
+def buildTag = System.getenv('VERSION') ?: 'dev'
+// Use task types
+task buildMyAppImage(type: DockerBuildImage) {
```

```suggestion
task buildMyAppImage(type: DockerBuildImage) {
```

nit: can we use a different name for these? Either `buildImage`, `buildDockerImage`, `buildDocker`?